web/src/stream_events.js

```diff
@@ -113,6 +116,8 @@ export function mark_subscribed(sub, subscribers, color) {
     if (sub.subscribed) {
         return;
     }
+    sub.push_notifications = push_notifications;
+    update_property(sub.stream_id, "push_notifications", sub.push_notifications);
```

Comment:
You can just call the `update_push_notifications_enabled` like we do for the `color` property, and we only call `update_property` on recieving event. Also, I think calling that function directly would be better for readability.

Reply:
Oh, sorry for the confusion. Ok, let's keep this as it is for now.